warhammer-core/WarhammerCore.Business/UserService.cs

```diff
@@ -9,17 +15,27 @@ namespace WarhammerCore.Business
     public class UserService : IUserService
     {
         private readonly IDataRepo _repo;
-        public UserService(IDataRepo repo)
+        private readonly IConfiguration _config;
+        public UserService(IDataRepo repo,
+            IConfiguration config
+            )
         {
             _repo = repo;
+            _config=config;
         }
 
         /// <summary>
         /// <inheritdoc/>
         /// </summary>
         public async Task<UserAuthInfo> SignInAsync(string email, string password)
         {
-            throw new NotImplementedException();
+          var user =   await _repo.GetUserByEmailAsync(email);
+
+            if (user.UserId == UserInfo.Empty.UserId) return null;
```

Comment:
We can also check the result by using `IsNullOrEmpty` method on UserInfo class.

```suggestion
            if (UserInfo.IsNullOrEmpty(user)) return null;
```